bounties/BA122401.md

```diff
@@ -0,0 +1,41 @@
+---
+code: BA122401
+title: Rocket Pool Support Payments
+proposal-author: epineph
+creation-date: 2024-05-28
+external-status: open
+summary: Support users within Rocket Pool's Discord support channel.
+reward: 
+  unit: USD
+  max: 54000
+  min: 54000
+repeatable: true
+expiry: August 31, 2024
+application-url: https://dao.rocketpool.net/t/round-12-call-for-bounty-applications-deadline-is-may-7/2919/2?u=shfryn
+skillsets:
+  - development
+  - protocol-knowledge
+  - node-operation
+contacts:
+  - ShfRyn
+  - epineph
+---
+
+## Summary 
+Provide support for questions posed by Rocket Pool community members in the # support discord channel; questions come largely from Node Operators and those hoping to be Node Operators. 
+
+## Verification
+
+Provide user support in the # support channel of discord, under the following terms:
+
+1) Provide at least 1 hour of work providing #support discord channel in EACH of the three months June-August 2024, as measured by RocketScrape; *alternatively provide a total 15 hours TOTAL over those 3 months.
+2) Within the first week of September 2024 a list of all potential awardees will be produced by GMC.
+3) A two week period will follow where any community member can publicly or privately notify the GMC that a potential awardee was not providing support (for example, was requesting support) or not providing quality support (for example, multiple ineffective messages/farming). *Anyone offering # support can also petition to add their name to the list if they narrowly failed the initial inclusion criteria.
+4) The GMC will investigate the # support record of any questioned potential awardee and designate either 0%, 50%, or 100% based on posts on # support channel. *The 50% level will be for users with approximately 25%-75% effective support messages. Potential awardees receiving 0% or 50% will be considered 0% or 50% for future # support bounties unless they specifically request and are granted re-evaluation.
+5) The “cost” below will be split proportionally amongst all awardees based on time spent as measure by RocketScrape and adjustments based on investigation in (4); payment will be done during the next GMC distribution phase.
+Note: No further # support retroactive awards will be considered for this timeframe to # support contributors.
+
+Payments portionally split amongst all awardees based on adjusted hours worked per RocketScrape.
```

Comment:
Like, why is 'this is how much you will get' close to the last line of the page. It's like one of the headline items.